packages/core/npm.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@cds/core",
+  "version": "5.2.1",
+  "license": "MIT",
+  "author": "clarity",
+  "description": "Clarity Design System Web Components",
+  "homepage": "https://clarity.design",
+  "keywords": ["web component", "design system", "clarity", "ui"],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vmware/clarity.git"
+  },
+  "bugs": {
+    "url": "https://github.com/vmware/clarity/issues"
+  },
+  "main": "./index.js",
+  "module": "./index.js",
+  "typings": "./index.d.ts",
+  "type": "module",
+  "files": ["*"],
+  "dependencies": {
+    "lit": "2.0.0-rc.2",
+    "ramda": "^0.27.1",
+    "tslib": "^2.2.0"
+  },
+  "optionalDependencies": {
+    "@cds/city": "^1.1.0",
+    "normalize.css": "^8.0.1",
+    "@types/resize-observer-browser": "^0.1.5"
+  }
+}
```

Comment:
This matches the same pattern as clr-icon and clr-ui. This makes it easier to have a clean package.json to publish rather than have to strip down the local package.json in the repo.